.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -30,4 +59,4 @@ Label | Description
 `templates-p1` | 1.0 MVP alpha release: Key doctypes that authors need help with.
 `templates-p2` | 2.0 Common doctypes used within open source.
 `templates-p3` | 3.0 Comprehensive set of doctypes
-`templates-p4` | 4.0 Special case doctypes
+`templates-p4` | 4.0 Special case doctype
```

Comment:
**Suggestion (non-blocking):** Let's also add that they need to contact one of the template leads to let them know their PR is ready for community review. I'll work on revamping our team page to make sure it's clear who the template leads are.

Reply:
I have included the below point as one of the tasks.
Contact the template leads to let them know that your draft is ready for community review.

Reply:
Perfect! Thank you!